DEBUG  [2018-08-03 16:53:21,961]  server.py  [223]  bind:  __________________З А П У С К ___ С Е Р В Е Р А ___________________________
INFO  [2018-08-03 16:53:21,962]  server.py  [224]  bind:  Запуск сервера с адрессом localhost:8888
INFO  [2018-08-03 16:53:23,873]  server.py  [234]  listen:  подключение с 127.0.0.1:1822
INFO  [2018-08-03 16:53:23,874]  server.py  [133]  connect_protocol:  отправленна проба на 127.0.0.1:1822 {'action': 'probe', 'time': '2018-08-03 16:53:23'}
INFO  [2018-08-03 16:53:23,874]  server.py  [136]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:1822  {'action': 'presence', 'time': '2018-08-03 16:53:23', 'type': 'status', 'account_name': 'Клиент', 'password': '2'}
ERROR  [2018-08-03 16:53:23,879]  server.py  [32]  save_name_from_presentce:  пользователь Клиент уже добавлен
INFO  [2018-08-03 16:53:24,000]  server.py  [40]  save_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-08-03 16:53:24,000]  server.py  [142]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:1822  {'response': 200, 'time': '2018-08-03 16:53:24', 'alert': 'Доступ разрешён'}
INFO  [2018-08-03 16:53:24,001]  server.py  [234]  listen:  подключение с 127.0.0.1:1821
INFO  [2018-08-03 16:53:24,001]  server.py  [133]  connect_protocol:  отправленна проба на 127.0.0.1:1821 {'action': 'probe', 'time': '2018-08-03 16:53:24'}
INFO  [2018-08-03 16:53:24,002]  server.py  [136]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:1821  {'action': 'presence', 'time': '2018-08-03 16:53:24', 'type': 'status', 'account_name': 'Пациент', 'password': '2'}
ERROR  [2018-08-03 16:53:24,005]  server.py  [32]  save_name_from_presentce:  пользователь Пациент уже добавлен
INFO  [2018-08-03 16:53:24,118]  server.py  [40]  save_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-08-03 16:53:24,118]  server.py  [142]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:1821  {'response': 200, 'time': '2018-08-03 16:53:24', 'alert': 'Доступ разрешён'}
INFO  [2018-08-03 16:53:24,119]  server.py  [150]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-08-03 16:53:24'}
INFO  [2018-08-03 16:53:24,119]  server.py  [185]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 1822)>
INFO  [2018-08-03 16:53:24,125]  server.py  [74]  get_contacts_list:  получен список контактов для: Клиент
INFO  [2018-08-03 16:53:24,125]  server.py  [190]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-08-03 16:53:24'} на сокет <socket.socket fd=344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 1822)>
INFO  [2018-08-03 16:53:24,325]  server.py  [150]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-08-03 16:53:24'}
INFO  [2018-08-03 16:53:24,325]  server.py  [185]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 1821)>
INFO  [2018-08-03 16:53:24,330]  server.py  [74]  get_contacts_list:  получен список контактов для: Пациент
INFO  [2018-08-03 16:53:24,330]  server.py  [190]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-08-03 16:53:24'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 1821)>
INFO  [2018-08-03 16:53:29,130]  server.py  [150]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-08-03 16:53:28', 'to': 'Клиент', 'from': 'Клиент', 'message': 'ligliglgi'}
INFO  [2018-08-03 16:53:29,130]  server.py  [171]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-08-03 16:53:28', 'to': 'Клиент', 'from': 'Клиент', 'message': 'ligliglgi'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 1821)>
INFO  [2018-08-03 16:53:32,530]  server.py  [150]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-08-03 16:53:32', 'to': 'Пациент', 'from': 'Пациент', 'message': 'uyfukufkuf'}
INFO  [2018-08-03 16:53:32,530]  server.py  [171]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-08-03 16:53:32', 'to': 'Пациент', 'from': 'Пациент', 'message': 'uyfukufkuf'} на сокет <socket.socket fd=344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 1822)>
INFO  [2018-08-03 16:53:34,730]  server.py  [154]  read_requests:  Клиент 332 ('127.0.0.1', 1821) отключился
INFO  [2018-08-03 16:53:35,530]  server.py  [154]  read_requests:  Клиент 344 ('127.0.0.1', 1822) отключился
DEBUG  [2018-09-09 20:57:47,566]  server.py  [237]  bind:  __________________З А П У С К ___ С Е Р В Е Р А ___________________________
INFO  [2018-09-09 20:57:47,659]  server.py  [238]  bind:  Запуск сервера с адрессом localhost:8888
ERROR  [2018-09-09 20:57:47,675]  server.py  [295]  <module>:  пользователь <built-in function all> уже добавлен
INFO  [2018-09-09 20:57:49,765]  server.py  [248]  listen:  подключение с 127.0.0.1:3614
INFO  [2018-09-09 20:57:49,765]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3614 {'action': 'probe', 'time': '2018-09-09 20:57:49'}
INFO  [2018-09-09 20:57:49,781]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3614  {'action': 'presence', 'time': '2018-09-09 20:57:49', 'type': 'status', 'account_name': 'Клиент', 'password': '2'}
ERROR  [2018-09-09 20:57:49,781]  server.py  [32]  save_name_from_presentce:  пользователь Клиент уже добавлен
INFO  [2018-09-09 20:57:49,843]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 20:57:49,843]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3614  {'response': 200, 'time': '2018-09-09 20:57:49', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 20:57:49,843]  server.py  [248]  listen:  подключение с 127.0.0.1:3613
INFO  [2018-09-09 20:57:49,859]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3613 {'action': 'probe', 'time': '2018-09-09 20:57:49'}
INFO  [2018-09-09 20:57:49,859]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3613  {'action': 'presence', 'time': '2018-09-09 20:57:49', 'type': 'status', 'account_name': 'Пациент', 'password': '2'}
ERROR  [2018-09-09 20:57:49,859]  server.py  [32]  save_name_from_presentce:  пользователь Пациент уже добавлен
INFO  [2018-09-09 20:57:49,859]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-09-09 20:57:49,874]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3613  {'response': 200, 'time': '2018-09-09 20:57:49', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 20:57:50,888]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 20:57:50'}
INFO  [2018-09-09 20:57:50,888]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 20:57:50'}
INFO  [2018-09-09 20:57:50,888]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3614)>
INFO  [2018-09-09 20:57:50,888]  server.py  [84]  get_contacts_list:  получен список контактов для: Клиент
INFO  [2018-09-09 20:57:50,888]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 20:57:50'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3614)>
INFO  [2018-09-09 20:57:50,888]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3613)>
INFO  [2018-09-09 20:57:50,888]  server.py  [84]  get_contacts_list:  получен список контактов для: Пациент
INFO  [2018-09-09 20:57:50,888]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 20:57:50'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3613)>
INFO  [2018-09-09 20:58:09,546]  server.py  [167]  read_requests:  Клиент 352 ('127.0.0.1', 3614) отключился
INFO  [2018-09-09 20:58:11,371]  server.py  [167]  read_requests:  Клиент 332 ('127.0.0.1', 3613) отключился
DEBUG  [2018-09-09 22:21:00,495]  server.py  [237]  bind:  __________________З А П У С К ___ С Е Р В Е Р А ___________________________
INFO  [2018-09-09 22:21:00,495]  server.py  [238]  bind:  Запуск сервера с адрессом localhost:8888
ERROR  [2018-09-09 22:21:00,495]  server.py  [295]  <module>:  пользователь <built-in function all> уже добавлен
INFO  [2018-09-09 22:21:09,122]  server.py  [248]  listen:  подключение с 127.0.0.1:3665
INFO  [2018-09-09 22:21:09,122]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3665 {'action': 'probe', 'time': '2018-09-09 22:21:09'}
INFO  [2018-09-09 22:21:09,122]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3665  {'action': 'presence', 'time': '2018-09-09 22:21:09', 'type': 'status', 'account_name': 'Пишущий', 'password': '2'}
ERROR  [2018-09-09 22:21:09,137]  server.py  [32]  save_name_from_presentce:  пользователь Пишущий уже добавлен
INFO  [2018-09-09 22:21:09,137]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Пишущий
INFO  [2018-09-09 22:21:09,137]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3665  {'response': 200, 'time': '2018-09-09 22:21:09', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 22:21:09,340]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 22:21:09'}
INFO  [2018-09-09 22:21:09,340]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3665)>
ERROR  [2018-09-09 22:21:09,340]  server.py  [87]  get_contacts_list:  у Пишущий нет контактов
INFO  [2018-09-09 22:21:09,340]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 22:21:09'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3665)>
INFO  [2018-09-09 22:21:54,970]  server.py  [167]  read_requests:  Клиент 352 ('127.0.0.1', 3665) отключился
INFO  [2018-09-09 22:21:58,558]  server.py  [248]  listen:  подключение с 127.0.0.1:3666
INFO  [2018-09-09 22:21:58,558]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3666 {'action': 'probe', 'time': '2018-09-09 22:21:58'}
INFO  [2018-09-09 22:21:58,574]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3666  {'action': 'presence', 'time': '2018-09-09 22:21:58', 'type': 'status', 'account_name': 'Пишущий', 'password': '2'}
ERROR  [2018-09-09 22:21:58,574]  server.py  [32]  save_name_from_presentce:  пользователь Пишущий уже добавлен
INFO  [2018-09-09 22:21:58,574]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Пишущий
INFO  [2018-09-09 22:21:58,574]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3666  {'response': 200, 'time': '2018-09-09 22:21:58', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 22:21:58,776]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 22:21:58'}
INFO  [2018-09-09 22:21:58,776]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=292, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3666)>
ERROR  [2018-09-09 22:21:58,776]  server.py  [87]  get_contacts_list:  у Пишущий нет контактов
INFO  [2018-09-09 22:21:58,776]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 22:21:58'} на сокет <socket.socket fd=292, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3666)>
INFO  [2018-09-09 22:22:10,336]  server.py  [167]  read_requests:  Клиент 292 ('127.0.0.1', 3666) отключился
DEBUG  [2018-09-09 22:22:28,432]  server.py  [237]  bind:  __________________З А П У С К ___ С Е Р В Е Р А ___________________________
INFO  [2018-09-09 22:22:28,432]  server.py  [238]  bind:  Запуск сервера с адрессом localhost:8888
ERROR  [2018-09-09 22:22:28,448]  server.py  [295]  <module>:  пользователь <built-in function all> уже добавлен
INFO  [2018-09-09 22:22:30,569]  server.py  [248]  listen:  подключение с 127.0.0.1:3668
INFO  [2018-09-09 22:22:30,569]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3668 {'action': 'probe', 'time': '2018-09-09 22:22:30'}
INFO  [2018-09-09 22:22:30,569]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3668  {'action': 'presence', 'time': '2018-09-09 22:22:30', 'type': 'status', 'account_name': 'Клиент', 'password': '2'}
ERROR  [2018-09-09 22:22:30,585]  server.py  [32]  save_name_from_presentce:  пользователь Клиент уже добавлен
INFO  [2018-09-09 22:22:30,601]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 22:22:30,601]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3668  {'response': 200, 'time': '2018-09-09 22:22:30', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 22:22:30,601]  server.py  [248]  listen:  подключение с 127.0.0.1:3669
INFO  [2018-09-09 22:22:30,601]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3669 {'action': 'probe', 'time': '2018-09-09 22:22:30'}
INFO  [2018-09-09 22:22:30,601]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3669  {'action': 'presence', 'time': '2018-09-09 22:22:30', 'type': 'status', 'account_name': 'Пациент', 'password': '2'}
ERROR  [2018-09-09 22:22:30,601]  server.py  [32]  save_name_from_presentce:  пользователь Пациент уже добавлен
INFO  [2018-09-09 22:22:30,616]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-09-09 22:22:30,616]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3669  {'response': 200, 'time': '2018-09-09 22:22:30', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 22:22:30,819]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 22:22:30'}
INFO  [2018-09-09 22:22:30,819]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 22:22:30'}
INFO  [2018-09-09 22:22:30,819]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3668)>
INFO  [2018-09-09 22:22:30,819]  server.py  [84]  get_contacts_list:  получен список контактов для: Клиент
INFO  [2018-09-09 22:22:30,819]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 22:22:30'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3668)>
INFO  [2018-09-09 22:22:30,819]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3669)>
INFO  [2018-09-09 22:22:30,819]  server.py  [84]  get_contacts_list:  получен список контактов для: Пациент
INFO  [2018-09-09 22:22:30,819]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 22:22:30'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3669)>
INFO  [2018-09-09 22:22:47,246]  server.py  [167]  read_requests:  Клиент 332 ('127.0.0.1', 3669) отключился
INFO  [2018-09-09 22:22:48,260]  server.py  [167]  read_requests:  Клиент 352 ('127.0.0.1', 3668) отключился
DEBUG  [2018-09-09 22:22:54,047]  server.py  [237]  bind:  __________________З А П У С К ___ С Е Р В Е Р А ___________________________
INFO  [2018-09-09 22:22:54,047]  server.py  [238]  bind:  Запуск сервера с адрессом localhost:8888
ERROR  [2018-09-09 22:22:54,047]  server.py  [295]  <module>:  пользователь <built-in function all> уже добавлен
INFO  [2018-09-09 22:22:56,138]  server.py  [248]  listen:  подключение с 127.0.0.1:3671
INFO  [2018-09-09 22:22:56,138]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3671 {'action': 'probe', 'time': '2018-09-09 22:22:56'}
INFO  [2018-09-09 22:22:56,138]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3671  {'action': 'presence', 'time': '2018-09-09 22:22:56', 'type': 'status', 'account_name': 'Клиент', 'password': '2'}
ERROR  [2018-09-09 22:22:56,138]  server.py  [32]  save_name_from_presentce:  пользователь Клиент уже добавлен
INFO  [2018-09-09 22:22:56,153]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 22:22:56,153]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3671  {'response': 200, 'time': '2018-09-09 22:22:56', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 22:22:56,169]  server.py  [248]  listen:  подключение с 127.0.0.1:3672
INFO  [2018-09-09 22:22:56,169]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3672 {'action': 'probe', 'time': '2018-09-09 22:22:56'}
INFO  [2018-09-09 22:22:56,169]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3672  {'action': 'presence', 'time': '2018-09-09 22:22:56', 'type': 'status', 'account_name': 'Пациент', 'password': '2'}
ERROR  [2018-09-09 22:22:56,185]  server.py  [32]  save_name_from_presentce:  пользователь Пациент уже добавлен
INFO  [2018-09-09 22:22:56,185]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-09-09 22:22:56,185]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3672  {'response': 200, 'time': '2018-09-09 22:22:56', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 22:22:56,185]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 22:22:56'}
INFO  [2018-09-09 22:22:56,185]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3671)>
INFO  [2018-09-09 22:22:56,200]  server.py  [84]  get_contacts_list:  получен список контактов для: Клиент
INFO  [2018-09-09 22:22:56,200]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 22:22:56'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3671)>
INFO  [2018-09-09 22:22:56,403]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 22:22:56'}
INFO  [2018-09-09 22:22:56,403]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3672)>
INFO  [2018-09-09 22:22:56,403]  server.py  [84]  get_contacts_list:  получен список контактов для: Пациент
INFO  [2018-09-09 22:22:56,403]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 22:22:56'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3672)>
INFO  [2018-09-09 22:23:12,861]  server.py  [50]  save_messages_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 22:23:12,861]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-09-09 22:23:12', 'to': 'Клиент', 'from': 'Клиент', 'message': 'мсмсммсмс'}
INFO  [2018-09-09 22:23:12,861]  server.py  [185]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-09-09 22:23:12', 'to': 'Клиент', 'from': 'Клиент', 'message': 'мсмсммсмс'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3672)>
INFO  [2018-09-09 22:23:20,770]  server.py  [50]  save_messages_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-09-09 22:23:20,770]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-09-09 22:23:20', 'to': 'Пациент', 'from': 'Пациент', 'message': 'мясм ясмсям'}
INFO  [2018-09-09 22:23:20,770]  server.py  [185]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-09-09 22:23:20', 'to': 'Пациент', 'from': 'Пациент', 'message': 'мясм ясмсям'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3671)>
INFO  [2018-09-09 22:24:08,241]  server.py  [50]  save_messages_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 22:24:08,241]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-09-09 22:24:08', 'to': 'Клиент', 'from': 'Клиент', 'message': 'рпчврвчрчврчрвчрч'}
INFO  [2018-09-09 22:24:08,241]  server.py  [185]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-09-09 22:24:08', 'to': 'Клиент', 'from': 'Клиент', 'message': 'рпчврвчрчврчрвчрч'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3672)>
INFO  [2018-09-09 22:24:29,332]  server.py  [167]  read_requests:  Клиент 332 ('127.0.0.1', 3672) отключился
INFO  [2018-09-09 22:24:30,549]  server.py  [167]  read_requests:  Клиент 352 ('127.0.0.1', 3671) отключился
DEBUG  [2018-09-09 23:32:57,313]  server.py  [237]  bind:  __________________З А П У С К ___ С Е Р В Е Р А ___________________________
INFO  [2018-09-09 23:32:57,313]  server.py  [238]  bind:  Запуск сервера с адрессом localhost:8888
ERROR  [2018-09-09 23:32:57,329]  server.py  [295]  <module>:  пользователь <built-in function all> уже добавлен
INFO  [2018-09-09 23:33:00,355]  server.py  [248]  listen:  подключение с 127.0.0.1:3755
INFO  [2018-09-09 23:33:00,355]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3755 {'action': 'probe', 'time': '2018-09-09 23:33:00'}
INFO  [2018-09-09 23:33:00,402]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3755  {'action': 'presence', 'time': '2018-09-09 23:33:00', 'type': 'status', 'account_name': 'Пациент', 'password': '2'}
ERROR  [2018-09-09 23:33:00,402]  server.py  [32]  save_name_from_presentce:  пользователь Пациент уже добавлен
INFO  [2018-09-09 23:33:00,418]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-09-09 23:33:00,418]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3755  {'response': 200, 'time': '2018-09-09 23:33:00', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 23:33:00,418]  server.py  [248]  listen:  подключение с 127.0.0.1:3754
INFO  [2018-09-09 23:33:00,418]  server.py  [144]  connect_protocol:  отправленна проба на 127.0.0.1:3754 {'action': 'probe', 'time': '2018-09-09 23:33:00'}
INFO  [2018-09-09 23:33:00,418]  server.py  [147]  connect_protocol:  получен презентс-сообщение от 127.0.0.1:3754  {'action': 'presence', 'time': '2018-09-09 23:33:00', 'type': 'status', 'account_name': 'Клиент', 'password': '2'}
ERROR  [2018-09-09 23:33:00,433]  server.py  [32]  save_name_from_presentce:  пользователь Клиент уже добавлен
INFO  [2018-09-09 23:33:00,449]  server.py  [40]  save_visit_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 23:33:00,449]  server.py  [153]  connect_protocol:  отправлен ответ на презентс 127.0.0.1:3754  {'response': 200, 'time': '2018-09-09 23:33:00', 'alert': 'Доступ разрешён'}
INFO  [2018-09-09 23:33:00,652]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 23:33:00'}
INFO  [2018-09-09 23:33:00,652]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'get_contacts', 'time': '2018-09-09 23:33:00'}
INFO  [2018-09-09 23:33:00,652]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3755)>
INFO  [2018-09-09 23:33:00,652]  server.py  [84]  get_contacts_list:  получен список контактов для: Пациент
INFO  [2018-09-09 23:33:00,652]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 23:33:00'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3755)>
INFO  [2018-09-09 23:33:00,652]  server.py  [199]  write_responses:  получен запрос на список контактов от сокета <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3754)>
INFO  [2018-09-09 23:33:00,667]  server.py  [84]  get_contacts_list:  получен список контактов для: Клиент
INFO  [2018-09-09 23:33:00,667]  server.py  [204]  write_responses:  отправлен ответ на запрос {'action': 'get_contacts', 'time': '2018-09-09 23:33:00'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3754)>
INFO  [2018-09-09 23:33:35,955]  server.py  [50]  save_messages_history:  добавлена новый запись в таблицу History: Клиент
INFO  [2018-09-09 23:33:35,955]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-09-09 23:33:35', 'to': 'Клиент', 'from': 'Клиент', 'message': 'ассмии'}
INFO  [2018-09-09 23:33:35,970]  server.py  [185]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-09-09 23:33:35', 'to': 'Клиент', 'from': 'Клиент', 'message': 'ассмии'} на сокет <socket.socket fd=352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3755)>
INFO  [2018-09-09 23:33:39,823]  server.py  [50]  save_messages_history:  добавлена новый запись в таблицу History: Пациент
INFO  [2018-09-09 23:33:39,823]  server.py  [163]  read_requests:  полученно входящее сообщение {'action': 'msg', 'time': '2018-09-09 23:33:39', 'to': 'Пациент', 'from': 'Пациент', 'message': 'вмвмвм'}
INFO  [2018-09-09 23:33:39,839]  server.py  [185]  write_responses:  сообщение отправленно {'action': 'msg', 'time': '2018-09-09 23:33:39', 'to': 'Пациент', 'from': 'Пациент', 'message': 'вмвмвм'} на сокет <socket.socket fd=332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8888), raddr=('127.0.0.1', 3754)>
INFO  [2018-09-09 23:38:36,536]  server.py  [167]  read_requests:  Клиент 332 ('127.0.0.1', 3754) отключился
INFO  [2018-09-09 23:38:38,361]  server.py  [167]  read_requests:  Клиент 352 ('127.0.0.1', 3755) отключился
